src/jumpstart.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
This JSON should be in the format of a .tsx file.

So instead of .json rename it to .tsx and on the first line prepend the phrase "export default"

Reply:
Also this file should be in src/pages/jumpstart